stylesheets/components/_button.scss

```diff
@@ -1,29 +1,35 @@
-// -----------------------------------------------------------------------------
-// This file contains all styles related to the button component.
-// -----------------------------------------------------------------------------
 button {
   border-radius: 7px;
   padding: 15px 25px;
   border: 1px solid;
   cursor: pointer;
 }
-.yellow-button {
-  background: $yellow-button-color;
+
+.sign-up-button {
+  background: $yellow-color;
   border-color: $white-color;
+  @include typography-weight600($light-grey, 16px, 24px);
 }
-.gray-button{
+
+.sign-in-button {
   background: $white-color;
-  border-color: $gray-button-color;
+  border-color: $gray-color;
+  @include typography-weight600($logo-color, 16px, 24px);
 }
+
 .playing-button {
   background: $white-color;
   border: 0;
-  color: $yellow-button-color;
+  color: $yellow-color;
+  @include typography-weight600($yellow-color, 16px, 24px);
+  text-decoration: underline;
```

Comment:
Too specific buttons.
Name buttons so they can be re-used. (Filled Button, Outline button, etc)
.playing-button - is a link, not a button.